.github/workflows/tests.yml

```diff
@@ -293,8 +297,6 @@ jobs:
         run: brew tests --online --coverage
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          # Set variables for coverage reporting.
-          HOMEBREW_CODECOV_TOKEN: 3ea0364c-80ce-47a3-9fba-93a940d4b5d7
```

Comment:
Need to submit coverage on macOS too (or: as well)

Reply:
@MikeMcQuaid makes sense, do you know what directory the coverage report is stored at?

Reply:
Previous runs were found at [/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/test/coverage/coverage.json](https://github.com/Homebrew/brew/runs/1875675486#step:7:34)

Reply:
oh I see, I think I read the CI yaml wrong. I added a commit that I think runs on macOS